src/rpc_client/chain_ops.rs

```diff
@@ -81,7 +83,7 @@ impl ApiInfo {
     }
 
     pub fn chain_export_req(params: ChainExportParams) -> RpcRequest<ChainExportResult> {
-        RpcRequest::new(CHAIN_EXPORT, params)
+        RpcRequest::new(CHAIN_EXPORT, params).with_timeout(Duration::MAX)
```

Comment:
It may be obvious, but I'd a note why it's needed here.

Reply:
Done